### DISCUSSION TITLE
Tokopedia: Cara membuat filter untuk hapus kolom iklan di aplikasi mobile

### DISCUSSION THREAD

**Discussion by 3xploiton3** [realodix/AdBlockID#724] [Q&A]
Bagaimana cara buat rule filter menghilangkan kolom yang saya lingkari,
Aplikasi Tokopedia di android

Kolom ini berisi iklan, sering tidak relevan dengan apa yang kita cari, sangat mengganggu

![](https://i.imgur.com/usSY4T5.jpg)

**Comment by realodix** [comment]
Saya kurang tahu deh bagaimana caranya di aplikasi Tokopedia Android.

Kalau di browser bisa pakai rule ini
`tokopedia.com###promo > .ta-product-container`

**Comment by 3xploiton3** [comment]
> Saya kurang tahu deh bagaimana caranya di aplikasi Tokopedia Android.
> 
> Kalau di browser bisa pakai rule ini
> `tokopedia.com###promo > .ta-product-container`

siap thx di browser berhasil tetapi di tokopedia android masih ada iklannya,
kenapa filter ini tidak dimasukkan ke database?

**Comment by realodix** [comment]
> kenapa filter ini tidak dimasukkan ke database?

Pada dasarnya Tokopedia merupakan situs tempat pihak ke 2 untuk mengiklankan produk. Hal ini yg membuat Saya ragu. 

Ada cukup banyak situs yg Saya masukkan whitelist di [AdBlockID/whitelist.adbl#L125](https://github.com/realodix/AdBlockID/blob/master/AdBlockID/whitelist.adbl#L125), karena mereka ya gitu, situs buat liat2 iklan.

**Comment by 3xploiton3** [comment]
@realodix saya rasa sebagai pengguna biasa, saya ke tokopedia untuk mencari sesuatu barang berdasarkan keyword yang saya cari, tetapi kolom iklan yg ditampilkan sama sekali tidak relevan, sangat menganggu apalagi di aplikasi tokopedia harus scroll2 kebawah, ini amat sering terjadi, 
menurut saya masuk ke kategori  Annoyance Ads,

Bila hanya iseng2 cari sesuatu di tampilan beranda tokopedia sdh cukup

**Comment by KagaKouko99** [comment]
ya klo market place ada iklan seperti itu ya wajar saja. mending diwhitelist saja khusus situs" seperti tokopedia

**Comment by 3xploiton3** [comment]
Contoh website amazon is a disana banyak banner2 iklan, saya lihat easylist juga block iklan2 disini,
Yang penting content original website masih terjaga

**Comment by realodix** [comment]
> Contoh website amazon is a disana banyak banner2 iklan, saya lihat easylist juga block iklan2 disini,
> Yang penting content original website masih terjaga

@3xploiton3 Sory bisa kasih contoh link Amazonnya gak? Saya cukup penasaran juga tentang Easylist block iklan / banner di Amazon secara spesifik?

**Comment by 3xploiton3** [comment]
contoh
https://www.amazon.com/s?k=i7+9700k&crid=32Z2X7JQ8AAX7&sprefix=i7+%2Caps%2C437&ref=nb_sb_ss_i_1_3

https://i.imgur.com/lwDAYck.png

**Comment by realodix** [comment]
@3xploiton3 Dari link dan SS yg Anda kirim, Saya tidak menemukan easylist blockir secara spesifik. Mereka blockir dan hide secara general.

Btw pertanyaan Anda tentang membuat rule sudah Saya jawab, dan untuk yg aplikasi Tokopedia Android, Anda bisa tanya pengembang aplikasi adblock yg Anda gunakan.

Sepertinya Saya bisa menutup issue ini. Thank you

**Comment by 3xploiton3** [comment]
@realodix maaf saya tanya lagi gimana filter hapus kolom iklan di browser mobile?

**Comment by realodix** [comment]
> siap thx di browser berhasil tetapi di tokopedia android masih ada iklannya,
> kenapa filter ini tidak dimasukkan ke database?

_Originally posted by @3xploiton3 in https://github.com/realodix/AdBlockID/issues/35#issuecomment-534607851_

Saya lihat request Anda di https://github.com/AdguardTeam/AdguardFilters/issues/44243 untuk blokir iklan pada pencarian di shoopee.co.id telah diterima.

Atas dasar itu, Saya buka kembali issue ini dan menambahkannya ke database.

**Comment by 3xploiton3** [comment]
kalau di adguard ada fitur opsi "Allow search ads and the self-promotion of websites" 
tapi kalau mengikuti rule easylist gak bisa

**Comment by realodix** [comment]
> kalau di adguard ada fitur opsi "Allow search ads and the self-promotion of websites"
> tapi kalau mengikuti rule easylist gak bisa

Jadi sebaiknya gimana?

**Comment by 3xploiton3** [comment]
Tambahkan saja rulenya, menurut saya iklan mengganggu, apalagi yg di aplikasi android,
